### DISCUSSION TITLE
Kindle highlights conversion?

### DISCUSSION THREAD

**Discussion by bard** [koreader/koreader#8746] [Q&A]
I have quite a few highlights/clippings on my (non-jailbroken) Kindle, but nowadays I do all my reading in KOReader on an Android-based e-reader and on my laptop. 

I'd like to import those highlights into KOReader. Looking at the "My Clippings.txt" format and at KOReader's sidecar format, the conversion doesn't seem trivial. Does any tool exist for that?

**Comment by Frenzie** [comment]
Probably not, unless it's something that KOHighlights does https://www.mobileread.com/forums/showthread.php?t=265433

**Comment by hius07** [comment]
Please note that "My Clippings.txt" stores only highlights/notes text.
More details are stored in the sdr folder near the book:
https://www.mobileread.com/forums/showthread.php?t=322172
Positions in kindle azw/kfx books are hard (if possible) to import to KOReader epub books.
If you need just the text of your notes, you can open "My Clippings.txt" in KOReader Text editor, copy the desired part of the text to clipboard, open the book and save copied text as a note. Not a quick way, sorry.
